client/verta/verta/operations/monitoring/monitored_entity.py

```diff
@@ -16,7 +16,38 @@
 
 
 class MonitoredEntity(entity._ModelDBEntity):
+    """A monitored entity persisted to Verta.
+
+    A monitored entity provides a named context to gather together data
+    summaries and alert configurations.
+
+    Attributes
+    ----------
+    name
+    workspace
+    alerts
+    """
+
     def __init__(self, conn, conf, msg):
+        """Intended for internal use only.
+
+        Users should obtain a monitored entity through the
+        :meth:`~verta.opertaions.monitoring.client.Client.get_or_create_monitored_entity`
+        method of the operations sub-client.
+
+        Parameters
+        ----------
+        conn
+            A connection object to the backend service.
+        conf
+            A configuration object used by conn methods.
+        msg
+            A protobuf message ai.verta.monitoring.MonitoredEntity
+        Returns
+        -------
+        MonitoredEntity
+            A monitored entity which is persisted by Verta.
+        """
```

Comment:
I'm not sure what `pydocstyle` has in mind with `D107`, but the [NumPy style guide](https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring) says `__init__()` is meant to be documented in the class docstring 🤔 I'm open to splitting them, but it's not a convention I've been following.

Reply:
That's probably correct and I'll adjust these things accordingly.